catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/ZipCompression.java

```diff
@@ -63,6 +63,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ZipCompression.class);
 
+    private static final String METACARD_PATH = "metacards/";
```

Comment:
Should the slash be File.separator?
Also, METACARD_PATH appears twice, should it be moved to a common constants location?